### PR TITLE
Desktop_Portfolio 

### PR DESCRIPTION
**Pull request by AhmadiGu**
I add a live demo to my Desktop_Portfolio  project it a responsive that is built with HTML and CSS and gets responsiveness by media queries

### CODE REVIEW

**Review by TadesseDev — APPROVED** [review]
### Hi @AhmadiGu 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve_happy

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  All linters passed 🥇
## Optional suggestions
- [Optional] You did well implementing the live demo @AhmadiGu, But Your pull request is made from `main` branch to `feature` branch. This is wrong according to the `GitHub` work flow. If you have deployed your app earlier, you should have opened an issue with a link to the online version of your app. I am only approving this pull request as the main requirement for this project is to check the live version of the project. Pleas make sure to use the right `GitHub` flow next time :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbhishekkM05** [comment]
good PR .

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 
 ## Live Demo 
-
```
Comment (MANYIKAM) [inline README.md:17]: I suuggest that you can add the live demo link

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@
 
 
 ## Live Demo 
- 
+
+ - GitHub: [live demo]( https://ahmadigu.github.io/Desktop_Portfolio/)
 
 ## Authors
```
Comment (MANYIKAM) [inline README.md:21]: May you please add author names who contributed in this project

**Review by MANYIKAM — COMMENTED** [review]
- [ ] Readme
- [ ] The following changes  has been suggested
- [ ] Update the authors 👍 
- [ ] update the live demo links
- [ ] Otherwise everything is fine
- [ ] 😄

**Pull request merged**